assembly/std/index.ts

```diff
@@ -60,11 +70,14 @@ export function localCall(
  *
  * Note: arguments serialization is to be handled by the caller and the callee.
  *
- * @param bytecode -
- * @param functionName -
- * @param args -
+ * @param bytecode - The bytecode of the contract containing the function to execute.
+ * @param functionName - The name of the function to call in that contract.
+ * @param args - The serialized arguments to send to the function to call.
```

Comment:
```suggestion
 * @param args - The arguments of the function we are calling.
```

(The "args" parameter doesn't need to be serialized. It is done in the "return env.localExecution(bytecode, functionName, args.serialize());")